spec/components/hiring_staff/vacancies_component_spec.rb

```diff
@@ -76,13 +94,43 @@
       it "renders the trust head office as a filter option" do
         expect(inline_component.css(".new_managed_organisations_form").to_html).to include("Trust head office")
       end
+
+      it "renders the open school as a filter option" do
+        expect(inline_component.css(".new_managed_organisations_form").to_html).to include("Open school")
+      end
+
+      it "does not render the closed school as a filter option" do
+        expect(inline_component.css(".new_managed_organisations_form").to_html).not_to include("Closed school")
+      end
     end
 
     context "when the organisation is a local authority" do
       let(:organisation) { create(:local_authority) }
+      let(:open_school) { create(:school, name: "Open school") }
+      let(:closed_school) { create(:school, :closed, name: "Closed school") }
       let!(:vacancy) { create(:vacancy, :published, :at_one_school) }
```

Comment:
Possibly change the vacancy to be `at_multiple_schools`, since the test is currently testing an impossible state (a listing associated directly to an LA)

```ruby
# line 117
vacancy.organisation_vacancies.create(organisation: organisation)
```